docs/reference/settings/security-settings.asciidoc

```diff
@@ -400,8 +400,9 @@ The path to the Java Keystore file that contains a private key and certificate.
 `ssl.key` and `ssl.keystore.path` may not be used at the same time.
 
 `ssl.keystore.type`::
-The format of the keystore file. Should be either `jks` to use the Java
```

Comment:
The reason for calling out PKCS11 in the settings is that the implementation now includes specific handling if this is set as the keystore.type ( As opposed to for instance `BCFKS` which can be used with the BouncyCastle FIPS Security Provider if configured, but handled transparently )